README.md

```diff
@@ -211,64 +262,68 @@ Alternatively, refactor the code to use `<img>` tags.
 
 ```html
 <div style="background-image: url(...)">
-  <span class='visually-hidden'>description of background image</span>
+  <span class="visually-hidden">description of background image</span>
 </div>
 ```
 
-### SVG
+### <a name="svg"></a>SVG
 
+When possible SVG images are prefered over bitmap. Some uses use screen enlargers to better view the content; SVG images scale losslessly while bitmap images become pixelated.
 
 1. Add a `<title>` node inside of the svg that describes the image.
-  
 
 ```html
 <svg>
   <title>description of svg</title>
 </svg>
 ```
 
-1. Hide the svg via `aria-hidden="true"` and add a visually hidden element that describes it.
+2. Hide the svg via `aria-hidden="true"` and add a visually hidden element that describes it.
```

Comment:
In markdown you can prefix each item in list with `1. ` and markdown will take care of enumerating it. This makes it easier to reorder lines without having to manually change the numbers in each line

Reply:
@aoxrud-ww I'd normally agree with that method but bc of the `html` blocks separating them they were both rendered as `1. `

<img width="494" alt="Screen Shot 2021-03-10 at 4 48 09 PM" src="https://user-images.githubusercontent.com/42351805/110702452-86311680-81c0-11eb-91fe-e906503232f8.png">